pkg/util/pbutil/pb.go

```diff
@@ -43,6 +47,7 @@ func ToProtoTimestamp(t time.Time) (tt *timestamp.Timestamp) {
 	tt, err := ptypes.TimestampProto(t)
 	if err != nil {
 		logger.Criticalf(nil, "Cannot convert time.Time [%+v] to ToProtoTimestamp[T]: %+v", t, err)
+		panic(err)
```

Comment:
no need to panic, Critical will send notification.

Reply:
udpated.

Reply:
done